Runtime/Scripts/RTCPeerConnection.cs

```diff
@@ -106,6 +106,26 @@ public RTCPeerConnectionState ConnectionState
             }
         }
 
+        /// <summary>
+        /// The readonly property of the <see cref="RTCPeerConnection"/> indicates
+        /// the current state of the peer connection by returning one of the
+        /// <see cref="RTCSignalingState"/> enum.
+        /// </summary>
+        /// <example>
+        /// <code>
+        /// var peerConnection = new RTCPeerConnection(configuration);
+        /// var signalingState = peerConnection.SignalingState;
+        /// </code>
+        /// </example>
+        /// <seealso cref="SignalingState"/>
```

Comment:
`seealso` is pointing itself. 🤔 
`RTCSignalingState` is correct?

Reply:
I was wrong about the `seealso` tag.
I think it's the `connectionState` that should be referenced (it's related this).
So I'll change it.